packages/components/tabs/src/Tabs.mdx

```diff
@@ -26,84 +26,46 @@ Be aware that the users will not see all the content at the same time. Make sure
 ```jsx
 // import { Tabs } from '@contentful/f36-tabs';
 
-<Tabs role="tablist">
-  <Tabs.Tab id="first" href="https://contentful.com" isSelected>
-    First
-  </Tabs.Tab>
-  <Tabs.Tab id="second" href="https://contentful.com">
-    Second
-  </Tabs.Tab>
-  <Tabs.Tab id="third" href="https://contentful.com">
-    Third
-  </Tabs.Tab>
+<Tabs defaultTab="first">
+  <Tabs.List variant="vertical-divider">
+    <Tabs.Tab id="first" href="https://contentful.com" isSelected>
```

Comment:
```suggestion
    <Tabs.Tab controls="first" href="https://contentful.com" isSelected>
```

Shouldn't this be `controls`?

Reply:
oh, thanks for catching it, this section needs to be updated